modules/scripts/startup-script/files/configure-ssh.yml

```diff
@@ -22,12 +22,16 @@
     bashrc: "{{ '/etc/bashrc' if ansible_facts['os_family'] == 'RedHat' else '/etc/bash.bashrc' }}"
     setup_ssh_script: "/bin/bash /usr/local/ghpc/setup-ssh-keys.sh"
   tasks:
+  - debug:
```

Comment:
nit: Ansible lint would complain here that 1. the task does not have a name and 2. you are not using the fully qualified task (`ansible.builtin.debug`). I think a name would be helpful if you are intending to leave this here as it will print in the logs.

Reply:
I removed it. Thanks